### PR TITLE
Bugfix: attribute type

### PR DESCRIPTION
**Pull request by Zorin95670**
Fix some case of https://github.com/ditrit/leto-modelizer-plugin-core/issues/188.

### CODE REVIEW

File: src/components/drawer/ComponentDetailPanel.vue [hunk 1]
```diff
@@ -193,6 +193,21 @@ async function save() {
   });
 }
 
+/**
+ * Get real type from definition.
```
Comment (k-y0u) [inline src/components/drawer/ComponentDetailPanel.vue:197]: Get type of component value. Can be possibly different from component type given in definition.

File: src/components/drawer/ComponentDetailPanel.vue [hunk 2]
```diff
@@ -193,6 +193,21 @@ async function save() {
...
+ * @param {ComponentAttributeDefinition} definition - Component attribute definition.
+ * @returns {string} Type of attribute.
+ */
+function getTypeFromDefinition(definition) {
```
Comment (k-y0u) [inline src/components/drawer/ComponentDetailPanel.vue:201]: proposition : getComponentValueType

File: changelog.md [hunk 3]
```diff
@@ -59,6 +59,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * Fix bug on using default file name from plugin instead of the file name specified by the user, when adding a component after creating a diagram from scratch.
 * Fix bug on deleted files while pushing a branch or using "upload to git" button, see [this issue](https://github.com/ditrit/leto-modelizer/issues/358).
 * Fix bug on import repository with `.git` or `/` at the end, see [this issue](https://github.com/ditrit/leto-modelizer/issues/368).
+* Fix bug on settings bad type on Link/Reference attribute, see [this issue](https://github.com/ditrit/leto-modelizer-plugin-core/issues/188).
```
Comment (k-y0u) [inline changelog.md:62]: setting

File: src/components/drawer/ComponentDetailPanel.vue (hunk 1 above)
Comment (Zorin95670) [reply src/components/drawer/ComponentDetailPanel.vue:197]: done

File: src/components/drawer/ComponentDetailPanel.vue (hunk 2 above)
Comment (Zorin95670) [reply src/components/drawer/ComponentDetailPanel.vue:201]: done

File: changelog.md (hunk 3 above)
Comment (Zorin95670) [reply changelog.md]: done

**Review by k-y0u — DISMISSED** [review]

**Pull request merged**